src/app/component/shared/secondary-menu/secondary-menu.component.spec.ts

```diff
@@ -119,7 +118,7 @@ describe("SecondaryMenuComponent", () => {
           actions: List<AnyMenuItem>([]),
           links: List<NavigableMenuItem>([]),
         },
-      } as PageInfoInterface);
+      } as IPageInfo);
```

Comment:
None of these type conversions are required. Unit tests could also use a re-write.